Resources/public/js/views/fields/ez-time-editview.js

```diff
@@ -57,14 +104,35 @@ YUI.add('ez-time-editview', function (Y) {
         _variables: function () {
             var def = this.get('fieldDefinition'),
                 field = this.get('field'),
-                time = '';
+                time = '',
+                hours = '',
+                minutes = '',
+                seconds = '',
+                isEmpty = true;
 
+            if (!this.get('supportsTimeInput')) {
```

Comment:
this can be simplified. No matter if the time input is supported, you always send 4 variables to the template `isRequired`, `supportsTimeInput`, `html5InputTime`, `useSeconds`. But you have to make sure that `useSeconds` is taken into account when formatting the `html5InputTime` value